models/engine/file_storage.py

```diff
@@ -71,3 +77,21 @@ def close(self):
             calls the reload() method for deserialization from JSON to objects
         """
         self.reload()
+
+    def get(self, cls, id):
+        """
+            a method to retrieve one object
+            cls: string representating the class name
+            id: string representating the object ID
+        """
+        cls_obj_str = "{}".format(id)
```

Comment:
"cls_obj_str" feels kind of wonky as a variable name, maybe something like "id_str"?